ironfish/src/telemetry/telemetry.ts

```diff
@@ -281,11 +281,24 @@ export class Telemetry {
   }
 
   submitNodeStarted(): void {
+    const timestamp = new Date()
+
     this.submit({
       measurement: 'node_started',
       fields: [{ name: 'online', type: 'boolean', value: true }],
-      timestamp: new Date(),
+      timestamp,
     })
+
+    if (this.metrics) {
+      this.submit({
+        measurement: 'node_started',
+        fields: [
+          { name: 'cpu_cores', type: 'integer', value: this.metrics.cpuCores },
+          { name: 'total_mem', type: 'integer', value: this.metrics.memTotal },
+        ],
+        timestamp,
+      })
+    }
```

Comment:
Can we use the existing telemetry above instead of duplicating it?

Reply:
Fixed.